src/map/clif.cpp

```diff
@@ -15744,7 +15744,7 @@ void clif_parse_Mail_getattach( int fd, struct map_session_data *sd ){
 
 				switch( pc_checkadditem(sd, item->nameid, item->amount) ){
 					case CHKADDITEM_NEW:
-						new_++;
+						new_ += itemdb_inventory_slot_needed(data, item->amount);
```

Comment:
shouldn't need here, if the attached item is GUID'd item with a stack of 10, player should get it as 10 in 1 inventory slot
```suggestion
						new_++;
```